lib/msal-common/src/cache/entities/CredentialEntity.ts

```diff
@@ -182,4 +192,16 @@ export class CredentialEntity {
     private static generateTargetForCacheKey(scopes?: string): string {
         return (scopes || "").toLowerCase();
     }
+
+    /**
+     * Generates scheme key component as per schema: <-scheme>
+     */
+    private static generateSchemeForCacheKey(credentialType: string, tokenType?: string): string {
```

Comment:
Any reason tokenType shouldn't be a required param here? Looks like you've already validated that its a non-empty string before this gets called

Reply:
Good catch, initially this was meant to be called for all credentials, most of which wouldn't have a tokenType and we'd just leave an extra trailing dash. After a lot of discussion/investigation we decided to limit the key change to AT with AuthScheme, which always has a tokenType, so now we can make it required.

Reply:
As I did the update I realized the whole method was unnecessary, we could just push the tokenType directly into the cache key elements array.